plugin.video.npr/addon.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.npr"
        name="NPR Music Videos"
-       version="4.0.0"
+       version="4.0.1"
        provider-name="t1m">
   <requires>
     <import addon="xbmc.python" version="3.0.0"/>
-    <import addon="script.module.t1mlib" version="4.0.4"/>
+    <import addon="script.module.t1mlib" version="4.0.6"/>
```

Comment:
just a heads up you don't need to bump this version everytime you do an update to the t1mlib unless there's functionality on the main addon that depends on functionality existing only on the new dependency version